### DISCUSSION TITLE
Build dashboard for ledger analysis & stats

### DISCUSSION THREAD

**Discussion by mistakia** [mistakia/nano-community#32] [Proposals]
**Ledger Stats Over Time (10mins, Hour, Day, Month)**
- [x] New addresses
- [x] Active addresses
- [x] Total Block Count
- [x] Send Block Count
- [x] Receive Block Count
- [x] Change Block Count
- [x] Open Block Count
- [x] Send Volume
- [x] Change Volume
- [x] Send amount (by ranges) (count and pct)
- [x] Addresses with balance (by ranges) (count and pct)
- [ ] Addresses in Profit
- [ ] Addresses in Loss
- [ ] Amount Deposited
- [ ] Amount Withdrawn 
- [ ] Number of addresses Depositing to an exchange
- [ ] Number of addresses Withdrawing from an exchange
- [ ] Number of accumulating addresses
- [ ] Balance of accumulating addresses
- [ ] Circulating Supply
- [ ] Long-Term Holder Supply
- [ ] Long-Term Holder Position Change
- [ ] Short-Term Holder Supply
- [ ] Tip Bot Number of Sends (by tipbot)
- [ ] Tip Bot Total Volume (by tipbot)
- [ ] Exchange Balance (by exchange)
- [ ] Exchange Inflow (by exchange)
- [ ] Exchange Outflow (by exchange)
- [ ] NVT Ratio / NVT Signal
- [ ] Velocity
- [ ] Supply Last Active
    - [ ] (<24h, 1d-1w, 1w-1m, 1m-3m, 6m-12m, 1y-2y, 2y-3y, 1y-3y, 3y+)
    
**Tasks**
- [ ] https://github.com/mistakia/nano-community/issues/108
- [x] feat: add download csv & json option to ledger charts (https://github.com/mistakia/nano-community/commit/2e5c1865020dc44d22a54b7dc583e7f1991f257a)
- [ ] bundle & export data for easy community analysis
  - [ ] initial distribution / landing account
  - [ ] exchanges deposits / withdrawals

**Comment by mistakia** [comment]
Basic page up — https://nano.community/ledger

**Comment by geommr** [comment]
Daily/weekly value transacted in fiat would be interesting (historical fiat price x total nano transacted in that period)
